self-hosted/README.md

```diff
@@ -327,6 +327,18 @@ export MYSQL_URL=mysql://<your-username>:<your-password>@aws.connect.psdb.cloud
 docker compose up
 ```
 
+### Database Names
+
+The database name that the Convex backend uses is equivalent to the instance name (replacing `-` with `_`). If no instance name is set, the Docker image defaults to `convex-self-hosted`, and the Convex backend will connect to the database `convex_self_hosted`.
+
+For the docker container, you can set the instance name via the `INSTANCE_NAME` envrionment variable.
+
```

Comment:
looks good. I am going to change the wording to be very clear that this is an example - since your example is specific to postgres, but the concept carries across any underlying persistence database.